File: java/dsframework/src/main/java/dsf/core/TaskNotExistException.java
package dsf.core;

/**
 *
 * @author arksea
 */
public class TaskNotExistException extends RuntimeException {

    public TaskNotExistException(String name) {
        super(name);
    }

    public TaskNotExistException(String name, Throwable ex) {
        super(name, ex);
    }
}

File: csharp/dsf/dsf/src/core/TaskNotExistException.cs
using System;

namespace com.baidu.dsf.core
{

    /// 
    /// <summary>
    /// @author arksea
    /// </summary>
    public class TaskNotExistException : Exception
    {

        public TaskNotExistException(string name) : base(name)
        {
        }

        public TaskNotExistException(string name, Exception ex) : base(name, ex)
        {
        }
    }

}